tests/contrib/zerorpc/zeropc_tests.py

```diff
@@ -36,16 +36,19 @@
 
 import pytest
 
+from elasticapm.conf.constants import ERROR
 from elasticapm.contrib.zerorpc import Middleware
 
-zerorpc = pytest.importorskip("zerorpc")
-gevent = pytest.importorskip("gevent")
 
```

Comment:
looks like isort doesn't like the two empty lines here, one should suffice :)

Reply:
gotcha (sorry forgot to install the precommit)

Reply:
No worries :) I'm mulling over how to make linting/autoformatting errors more visible anyway (see https://twitter.com/beniwohli/status/1168511125949943814 :D)